client/components/payment-method-required-for-oc-pill/index.js

```diff
@@ -0,0 +1,96 @@
+/* global wc_stripe_settings_params */
+import { __, sprintf } from '@wordpress/i18n';
+import React from 'react';
+import styled from '@emotion/styled';
+import interpolateComponents from 'interpolate-components';
+import { Icon, info } from '@wordpress/icons';
+import Popover from 'wcstripe/components/popover';
+import {
+	PAYMENT_METHOD_AFFIRM,
+	PAYMENT_METHOD_KLARNA,
+} from 'wcstripe/stripe-utils/constants';
+
+const StyledPill = styled.span`
+	display: inline-flex;
+	align-items: center;
+	gap: 4px;
+	padding: 4px 8px;
+	border: 1px solid #fcf9e8;
+	border-radius: 2px;
+	background-color: #fcf9e8;
+	color: #674600;
+	font-size: 12px;
+	font-weight: 400;
+	line-height: 16px;
+	width: fit-content;
+`;
+
+const StyledLink = styled.a`
+	&:focus,
+	&:visited {
+		box-shadow: none;
+	}
+`;
+
+const IconWrapper = styled.span`
+	height: 16px;
+	cursor: pointer;
+`;
+
+const AlertIcon = styled( Icon )`
+	fill: #674600;
+`;
+
+const IconComponent = ( { children, ...props } ) => (
+	<IconWrapper { ...props }>
+		<AlertIcon icon={ info } size="16" />
+		{ children }
+	</IconWrapper>
+);
+
+const PaymentMethodUnavailableDueConflictPill = ( { id, label } ) => {
```

Comment:
I think this file got mixed with `client/components/payment-method-unavailable-due-conflict-pill/index.js`, the changes in that file should be moved here.